db/dataset_eval.py

```diff
@@ -283,8 +295,12 @@ def _create_job(connection, dataset_id, normalize, eval_location, filter_type=No
         raise ValueError("Incorrect 'eval_location'. Must be one of %s" % VALID_EVAL_LOCATION)
     snapshot_id = db.dataset.create_snapshot(dataset_id)
     query = sqlalchemy.text("""
-                INSERT INTO dataset_eval_jobs (id, snapshot_id, status, options, eval_location)
-                     VALUES (uuid_generate_v4(), :snapshot_id, :status, :options, :eval_location)
+                INSERT INTO dataset_eval_jobs (id, snapshot_id, status, options, eval_location,
```

Comment:
This query could be formatted better: https://github.com/metabrainz/guidelines/blob/master/Python.md#sql


I don't think we mind lines of ~130 characters so this could work like

```
INSERT INTO dataset_eval_jobs (id, snapshot_id, status, options, eval_location, _value, gamma_value, preprocessing_values)
     VALUES (uuid_generate_v4(), :snapshot_id, :status, :options, :eval_location, :c_value, :gamma_value, :preprocessing_values)
  RETURNING id
```

Example: https://github.com/metabrainz/critiquebrainz/blob/master/critiquebrainz/db/review.py#L295